sources/packages/api/src/services/student/student.service.ts

```diff
@@ -177,4 +177,66 @@ export class StudentService extends RecordDataModelService<Student> {
       this.repo.save(studentToUpdate);
     }
   }
+
+  /**
+   * Update the PD Sent Date
+   * @param studentId
+   */
+  async updatePDSentDate(studentId: number): Promise<Student> {
+    // get the Student Object
+    const studentToUpdate = await this.repo.findOneOrFail({
+      id: studentId,
+    });
+    if (studentToUpdate) {
+      const now = new Date();
+      // Date in UTC
+      studentToUpdate.studentPDSentAt = new Date(
+        now.getUTCFullYear(),
+        now.getUTCMonth(),
+        now.getUTCDay(),
+        now.getUTCHours(),
+        now.getUTCMinutes(),
+        now.getUTCSeconds(),
+      );
+      return this.repo.save(studentToUpdate);
+    }
+  }
+
+  /**
+   * Update the PD Sent Date
+   * @param studentId
+   * @param status
+   */
+  async updatePDStatusNDate(
+    studentId: number,
+    status: boolean,
+  ): Promise<Student> {
+    // get the Student Object
+    const studentToUpdate = await this.repo.findOneOrFail({
+      id: studentId,
+    });
+    if (studentToUpdate) {
+      const now = new Date();
+      studentToUpdate.studentPDVerified = status;
+      // Date in UTC format
+      studentToUpdate.studentPDUpdateAt = new Date(
+        now.getUTCFullYear(),
```

Comment:
Please use the dayjs library already in API to do this type of manipulation: https://day.js.org/docs/en/plugin/utc